### DISCUSSION TITLE
Black screen wt32-sc01_4MB V0.6.3

### DISCUSSION THREAD

**Discussion by alexval2014** [HASwitchPlate/openHASP#328] [Q&A] [answered]
Hello. As soon as one problem was solved for version 0.6.3, another one appeared. The code has compiled.
The screen on the board after downloading the firmware is dark. There are no attempts to connect to wifi or create your own access point. On version 0.6.2 everything works.
Why did I start trying version 0.6.3 instead of 0.6.2 because I can't turn on the Cyrillic font. Although I did everything according to the instructions.
```
[platformio]
extra_configs =
    ; Uncomment or edit the lines to show more User Setups in `the` PIO sidebar
     user_setups/esp32/*.ini

[override]
; -- Hasp config options --------------------------------------
build_flags =
; -- Uncomment the next line to use the file include/user_config_override.h settings
    -DUSE_CONFIG_OVERRIDE

;region -- Default Build Environments : Used when Build All ---
extra_default_envs =
    ; Uncomment specific environments or create extra:
      wt32-sc01_4MB
    ; wt32-sc01_16MB
;endregion

;region -- Define your local COM ports for each environment ---
[env:wt32-sc01_4MB]
monitor_port = COM12      ; Change to the correct port
upload_port = ${env:wt32-sc01_4MB.monitor_port}
```

**Comment by alexval2014** [comment]
my board's response is wt32-sc01_4MB
`
ets Jun  8 2016 00:22:57

rst:0x10 (RTCWDT_RTC_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
configsip: 0, SPIWP:0xee
clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
mode:DIO, clock div:2
load:0x3ffe6100,len:124
ho 0 tail 12 room 4
load:0x4004c000,len:2020
load:0x40050000,len:9668
entry 0x4004c15c
Fatal exception (28): LoadProhibited
epc1=0x4004c1a7, epc2=0x00000000, epc3=0x00000000, excvaddr=0x00130570, depc=0x00000000

`

**Reply by kamiKAC** [reply]
Hi
Same problem here:
rst:0x10 (RTCWDT_RTC_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
configsip: 0, SPIWP:0xee
clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
mode:DIO, clock div:2
load:0x3ffe6100,len:124
ho 0 tail 12 room 4
load:0x4004c000,len:2044
load:0x40050000,len:9668
entry 0x4004c15c
Fatal exception (28): LoadProhibited
epc1=0x4004c1a7, epc2=0x00000000, epc3=0x00000000, excvaddr=0x00130570, depc=0x00000000

Tried precompiled versions:
wt32-sc01_full_4MB_v0.6.3-dev_8184d2c.bin
wt32-sc01_full_4MB_v0.6.3-dev_8ab5c9a.bin
wt32-sc01_full_4MB_v0.6.3-dev_cfdeaaa.bin

Best Regards

**Comment by fvanroie** [comment, marked as answer]
Thanks for reporting this issue. Indeed there was an error in the `user_setups\esp32\wt32-sc01.ini` file. 71f01ef
This has now been fixed and I confirmed it is working on my local test device. Please pull the latest source and try again.

**Reply by kamiKAC** [reply]
Works fine. Also solves my problem with latin2 font not fitting into 4MB flash device.
Thanks for the prompt bugfix.

Awesome project.

**Reply by fvanroie** [reply]
Thanks for confirming the fix!

**Reply by alexval2014** [reply]
Thanks, it helped me too.

**Reply by fvanroie** [reply]
Cool, thanks for reporting back. Sorry for the late reply, I had to dig up and test it here locally to find the issue.

**Comment by jor3l** [comment]
Hello, I'm new here, just got the device, downloaded the latest version from the web (16mb one), flashed using ESPTools and the screen is black. Any ideas as to why or how to debug? Thanks!

**Reply by jor3l** [reply]
Update: Compiled using VSCode and now it works so there is something wrong with the bin

**Reply by fvanroie** [reply]
Can you please specify the full filename which has the issue? Thanks.

**Reply by jor3l** [reply]
> Can you please specify the full filename which has the issue? Thanks.

Tried wt32-sc01_full_4MB_v0.6.3_497f611 and wt32-sc01_full_16MB_v0.6.3_497f611
